x-pack/plugins/cases/public/components/case_view/translations.ts

```diff
@@ -137,30 +137,63 @@ export const DOES_NOT_EXIST_BUTTON = i18n.translate('xpack.cases.caseView.doesNo
 });
 
 export const TOTAL_ALERTS_METRIC = i18n.translate('xpack.cases.caseView.metrics.totalAlerts', {
-  defaultMessage: 'Total Alerts',
+  defaultMessage: 'Total alerts',
 });
 
 export const ASSOCIATED_USERS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.associatedUsers',
   {
-    defaultMessage: 'Associated Users',
+    defaultMessage: 'Associated users',
   }
 );
 
 export const ASSOCIATED_HOSTS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.associatedHosts',
   {
-    defaultMessage: 'Associated Hosts',
+    defaultMessage: 'Associated hosts',
   }
 );
 
 export const ISOLATED_HOSTS_METRIC = i18n.translate('xpack.cases.caseView.metrics.isolatedHosts', {
-  defaultMessage: 'Isolated Hosts',
+  defaultMessage: 'Isolated hosts',
 });
 
 export const TOTAL_CONNECTORS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.totalConnectors',
   {
-    defaultMessage: 'Total Connectors',
+    defaultMessage: 'Total connectors',
   }
 );
+
+export const CASE_CREATED = i18n.translate('xpack.cases.caseView.metrics.lifespan.caseCreated', {
+  defaultMessage: 'Case created',
```

Comment:
Yeah good point, I wonder if we should switch the list page to say `Created on` 🤔  because technically you can reopen a case.

@monina-n @nastasha-solomon what do you think?

Reply:
I'm leaning towards changing the list page from `Opened on` to `Created on` for the reasons @jonathan-buttner listed. A case can be opened multiple times, but it can only be made (created) once.

Reply:
Sounds good! I'll switch it to `Created on` 👍